src/lib/components/pages/settings/giftcards/settings.svelte

```diff
@@ -183,23 +257,132 @@
 
 <Modal
 	open={openBulkIssueModal}
-	header="Bulk issue"
+	header="Bulk Issue Gift Cards"
 	onCancel={() => (openBulkIssueModal = false)}
 	onClose={() => (openBulkIssueModal = false)}
 	onOk={handleBulkIssue}
 	disableElements={loading}
 >
-	<div>Bulk issue</div>
+	<div class="space-y-2">
+		<Input
+			size="sm"
+			placeholder="Cards issued"
+			label="Cards Issued"
+			class="flex-1"
+			bind:value={cardsIssued}
+			disabled={loading || disabled}
+			required
+		/>
+		<div class="flex items-start gap-2">
+			<Input
+				size="sm"
+				type="number"
+				min={1}
+				placeholder="Set balance amount"
+				label="Balance amount"
+				class="flex-2/3"
+				bind:value={balanceAmount}
+				disabled={loading || disabled}
+				required
+			/>
+			<ShopCurrenciesSelect
+				label="Currency"
+				class="flex-1"
```

Comment:
please do research on what do flex-1, flex-1/2, ... do
and consider changing them, for this 2 input fields